### ISSUE TITLE
Can not set RealTimeChannelOptions params 

### ISSUE THREAD

**Issue by 1N50MN14** [ably/ably-flutter#182]
Because cipher is not nullable I'm unable to pass `{'rewind':'1'}` to `RealtimeChannelOptions`  without using cipher.

```javascript
RealtimeChannelOptions RealtimeChannelOptions(
  Object cipher, {
  Map<String, String>? params,
  List<ChannelMode>? modes,
})
```

Otherwise I could not find a reference within the flutter example how to use one...

Upon further investigation, `CipherParams` seems to be abstract. Crypto interface doesn't seem to be implemented so I couldn't use `getDefaultParams` using my server-side generated key. So I tried to manually pass `cipherParams` as a Map but that threw a methodChannel error as well.

Basically locked out of symmetric encryption as well as channel options...

**Comment by ben-xD** [comment]
Thanks for reporting this @1N50MN14. I'm working on the fix for this: to let you use channel options properly. The symmetric encryption feature is currently not implemented, and you should not be forced to set `cipher`. However, channel options (Params and mode) **should** already be working. Unfortunately, in Ably-java they are both not handled correctly at the moment.

In Ably-java: Unfortunately, the params passed by you (the user) is used as CipherParams instead, and the modes is simply not set.

In Ably-cocoa: It looks like this should work fine. 

> So I tried to manually pass cipherParams as a Map but that threw a methodChannel error as well.

I'm not sure what you mean by passing `cipherParams` manually, can you clarify?
